Makefile

```diff
@@ -0,0 +1,18 @@
+BINARY_FOLDER_PREFIX?=/usr/local
```

Comment:
Why the `?` ?

Reply:
It's the conditional variable assignment thing. We'll pass the `#{prefix}` from the Homebrew script to our Makefile. This is just saying if we don't pass it in, we default to `/usr/local`.